src/workerd/io/worker.c++

```diff
@@ -2249,6 +2322,20 @@ kj::Promise<void> Worker::Isolate::attachInspector(
     kj::HttpHeaderId controlHeaderId) const {
   KJ_REQUIRE(impl->inspector != nullptr);
 
+  kj::HttpHeaders headers(headerTable);
+  headers.set(controlHeaderId, "{\"ewLog\":{\"status\":\"ok\"}}");
```

Comment:
Whoops, I guess we should move this out of the workerd code. (Not necessarily in this PR.)

Reply:
Yep, agreed. Not going to worry about it here tho.